roles/ansible-service-broker-setup/defaults/main.yml

```
@@ -12,7 +12,11 @@ broker_mobile_configuration:
   value: true
 
 #The url for the registry that hosts our APB images
+ansible_playbookbundle_registry_type: dockerhub
 ansible_playbookbundle_registry_url: https://registry.hub.docker.com
-
 #The organisation that our APB images are in
 ansible_playbook_bundle_registry_org: aerogearcatalog
+rhcc_registry_auth_name: ""
+rhcc_registry_auth_type: ""
```

What should these 2 be set to? Should we default to what the openshift-ansible installer sets? https://github.com/openshift/openshift-ansible/blob/release-3.11/roles/ansible_service_broker/tasks/install.yml#L200..L201

```suggestion
rhcc_registry_auth_type: "secret"
```

---

I think it's good idea to refer to the secret created by default. Just verified that it's present on my cluster. `<openshift-url>/console/project/openshift-ansible-service-broker/browse/secrets/asb-registry-auth`